package.json

```diff
@@ -41,8 +43,13 @@
     "babelify": "^7.3.0",
     "browserify": "^13.0.0",
     "chai": "^3.5.0",
+    "chai-as-promised": "^6.0.0",
     "diff": "^3.2.0",
+    "eslint": "^3.18.0",
```

Comment:
I wasn't sure if you could define .eslintrc files by using the `standard` rep so I was doing it backwards by extending the standard-plugin, but yeah I can reverse that.